### ISSUE TITLE
omc doesn't report errors associated with importing of packages

### ISSUE THREAD

**Issue by jdpipe** [OpenModelica/OpenModelica#10046]
### Description

With OM 1.20.0 on Ubuntu 20.04, `omc` provides no helpful output for the case where a model attempts to import a model that contains syntax errors. For example:

**Sauce.mo**
```
model Sauce
	import MyPkg;
	MyPkg.Example here;
	Real x,y;
equation
	x = 5;
	here.x = x;
	here.y = y;
end Sauce;
```

**stuff/MyPkg/package.mo**
```
within;
package MyPkg
  annotation(uses(Modelica(version="3.2.3")));
end MyPkg;
```

**stuff/MyPkg/package.order**
```
Example
```

**stuff/MyPkg/Example.mo**
```
within MyPkg;
model Example
	import Modelica.Math.sin;
	Real x,y;
//equation
	y = sin(x);
end Example;
```
In this case, the output from `OPENMODELICALIBRARY=~/.openmodelica/libraries:$PWD/stuff omc Sauce.mo` is:

```
Error processing file: Sauce.mo
[/home/john/testomc/Sauce.mo:2:2-2:14:writable] Error: Import MyPkg not found in scope <top>.
[/home/john/testomc/Sauce.mo:3:2-3:20:writable] Error: Class MyPkg.Example not found in scope Sauce.
Error: Internal error Instantiation of Sauce failed with no error message.

# Error encountered! Exiting...
# Please check the error message and the flags.

Execution failed!
```

Which is not helpful, since it is not clear if there is an error in `Sauce.mo` or something inside of `MyPkg`. In fact package `MyPkg` *was* found here, but could not be imported due to an error in the file `stuff/MyPkg/Example.mo`. If the line `//equation` from that file is uncommented (as it should be), then the output is as it should be:

```
class Sauce
  Real here.x;
  Real here.y;
  Real x;
  Real y;
equation
  here.y = sin(here.x);
  x = 5.0;
  here.x = x;
  here.y = y;
end Sauce;
Notification: Automatically loaded package Modelica 3.2.3 due to uses annotation from MyPkg.
Notification: Automatically loaded package Complex 4.0.0 due to uses annotation from Modelica.
Notification: Automatically loaded package ModelicaServices 4.0.0 due to uses annotation from Modelica.
Notification: Automatically loaded package MyPkg (default) due to usage.
```

Are there flags I can use with the `omc` command that would give me the kind of flow-through error reporting that I would expect to see here? I tried `--showErrorMessages` but it just repeats stuff that is already being output.

Clearly `omc` is not normally used in this kind of 'bare' way, and normally gets called programmatically via other tools. However, I wonder if the implementation could be changed to provide more useful feedback to the user by default, and then for such output to be suppressed explicitly, if needed, when `omc` is invoked those other ways?

I note that if I create an `.mos` script with `loadModel(Sauce);getErrorString();`, then I get even less output, just `false` and `""`.

See also #10043.

**Comment by jdpipe** [comment]
Another interesting issue is that there is no error raised here if the line `import Modelica.Math.sin;` is commented out. In that case, the function `sin` is not defined, but `omc` doesn't report the error.

**Comment by perost** [comment]
> Are there flags I can use with the `omc` command that would give me the kind of flow-through error reporting that I would expect to see here? I tried `--showErrorMessages` but it just repeats stuff that is already being output.

No, `--showErrorMessages` should show all errors, so in this case it seems like no error is actually issued. I'm not sure why there's no error issued in this case, there really should be. I'll have to investigate a bit.

>Another interesting issue is that there is no error raised here if the line import Modelica.Math.sin; is commented out. In that case, the function sin is not defined, but omc

`sin` is a builtin function in Modelica, so it's always defined. `Modelica.Math.sin` is just a wrapper for `sin` that provides e.g. graphical annotations so it can be used in a drag-and-drop editor like OMEdit.

**Comment by casella** [comment]
@perost the error reported as
```
[/home/john/testomc/Sauce.mo:2:2-2:14:writable] Error: Import MyPkg not found in scope <top>.
```
where the cause is that MyPkg cannot be loaded is really misleading. "Not found" is not the same as "Invalid" or "Cannot be parsed" or something. Can you fix that?
